### PR TITLE
Cisco specific: Check debug shell status before ecn marking test

### PR DESCRIPTION
**Pull request by sreejithsreekumaran**
### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Back port request
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311
- [x] 202405

#### What is the motivation for this PR?

Testcase execution might fail if a previous test did a config reload
which results in delay of dshell init which is expected.

#### How did you do it?

Added a pytest fixture to run along with testcase.

#### How did you verify/test it?

on Snappi based run

### CODE REVIEW

**Review by alpeshspatel — APPROVED** [review]

**Comment by alpeshspatel** [comment]
fyi: @abdosi  @sdszhang This fixes the ixia ECN test case

File: tests/snappi_tests/files/helper.py [hunk 1]
```diff
@@ -151,3 +151,38 @@ def revert_config_and_reload(node, results=None):
...
+
+    if is_cisco_device(rx_duthost):
+        dutport = snappi_ports[0]['peer_port']
+        asic_namespace_string = ""
+        syncd_string = "syncd"
+        if rx_duthost.is_multi_asic:
+            asic = rx_duthost.get_port_asic_instance(dutport)
+            asic_namespace_string = " -n " + asic.namespace
+            asic_id = rx_duthost.get_asic_id_from_namespace(asic.namespace)
+            syncd_string += asic_id
```
Comment (rraghav-cisco) [inline tests/snappi_tests/files/helper.py:169]: @sreejithsreekumaran : This should be:
syncd_string += str(asic_id)

Comment (sreejithsreekumaran) [reply tests/snappi_tests/files/helper.py]: done

**Review by sdszhang — APPROVED** [review]
LGMT

**Pull request merged**